packages/circuit-ui/components/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -124,48 +139,56 @@ export const RadioButtonGroup = forwardRef(
         'The `label` prop is missing. Pass `hideLabel` if you intend to hide the label visually.',
       );
     }
+
+    if (isEmpty(options)) {
+      return null;
+    }
```

Comment:
This isn't a breaking change, right? It's super unlikely that anyone used the component without options (that would make no sense), but I'm not sure if there could be edge cases like rendering the component and then fetching options somehow, where this change might be breaking.